ui_advanced_config.py

```diff
@@ -220,7 +280,74 @@ def BuildUI(self):
             padx=10,
             pady=10,
         )
+        self.componentsCreateButtom.grid(
+            row=15,
+            column=1,
+            columnspan=2,
+            sticky="nsew",
+            ipadx=2,
+            ipady=2,
+            padx=10,
+            pady=10,
+        )
+        self.componentsDeleteButton.grid(
+            row=16,
+            column=1,
+            columnspan=2,
+            sticky="nsew",
+            ipadx=2,
+            ipady=2,
+            padx=10,
+            pady=10,
+        )
+
+        # Make Object Widgets
+        self.selectObjectsCombo = uiComboBox(master=self.objectsColumn)
+        self.selectObjectsCombo.configure(state="readonly")
+        self.objectsLabel = uiLabel(master=self.objectsColumn, text="Objects")
+        self.objectsUpdateButton = uiButton(
+            master=self.objectsColumn, command=self.update_objects_json, text="Update"
+        )
+        self.objectsCreateButton = uiButton(
+            master=self.objectsColumn, command=self.create_object, text="Create"
+        )
+        self.objectsDeleteButton = uiButton(
+            master=self.objectsColumn, command=self.delete_object, text="Delete"
+        )
+        self.objectsIDLabel = uiLabel(master=self.objectsColumn, text="ID:")
+        self.objectsIDEntry = uiEntry(master=self.objectsColumn)
+        self.objectsNameLabel = uiLabel(master=self.objectsColumn, text="Name:")
+        self.objectsNameEntry = uiEntry(master=self.objectsColumn)
+        self.objectsFillAliveLabel = uiLabel(
+            master=self.objectsColumn, text="Fill Alive:"
+        )
+        self.objectsFillAliveEntry = uiEntry(
+            master=self.objectsColumn,
+        )
+        self.objectsFillDeadLabel = uiLabel(
+            master=self.objectsColumn, text="Fill Dead:"
+        )
+        self.objectsFillDeadEntry = uiEntry(master=self.objectsColumn)
+        self.objectsTextLabel = uiLabel(master=self.objectsColumn, text="Text:")
+        self.objectsTextEntry = uiEntry(master=self.objectsColumn)
+        self.objectsHealthLabel = uiLabel(master=self.objectsColumn, text="Health:")
+        self.objectsHealthEntry = uiEntry(master=self.objectsColumn)
+        self.objectsHealthEntry.config(
+            validate="all", validatecommand=(self.validateNum, "%P")
+        )
+        self.objectsDensityLabel = uiLabel(master=self.objectsColumn, text="Density:")
+        self.objectsDensityEntry = uiEntry(master=self.objectsColumn)
+        self.objectsDensityEntry.config(
+            validate="all", validatecommand=(self.validateNum, "%P")
+        )
+        self.objectsCompIDsLabel = uiLabel(master=self.objectsColumn, text="Comp IDs:")
+        self.objectsCompIDsCombo = uiComboBox(master=self.objectsColumn)
+        self.objectsPointsCountLabel = uiLabel(
+            master=self.objectsColumn, text="Points Count:"
+        )
+        self.objectsPointsCountEntry = uiEntry(master=self.objectsColumn)
```

Comment:
You mentioned wanting to make this a drop down. It's likely this is still on your radar, but this is a reminder if it's been at all forgotten

Reply:
I haven't forgotten but at this point I kinda want to introduce it in a new PR. This one is huge.

Reply:
Makes sense to me!